examples/lighting-app/nrfconnect/README.md

```diff
@@ -1,289 +1,333 @@
+        $ export ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
+        $ export GNUARMEMB_TOOLCHAIN_PATH=toolchain-dir
 
-After your environment is set up, you are ready to build the example. The
-recommended tool for building and flashing the device is
-[west](https://docs.zephyrproject.org/latest/guides/west/).
+Now you can proceed with the [Building](#building) instruction.
 
-The following commands will build the `lighting-app` example:
+<hr>
 
-        $ cd ~/connectedhomeip/examples/lighting-app/nrfconnect
+<a name="building"></a>
 
-        # If this is a first time build or if `build` directory was deleted
-        $ west build -b nrf52840dk_nrf52840
+## Building
 
-        # Any subsequent build
-        $ west build
+Complete the following steps, regardless of the method used for setting up the environment:
 
-After a successful build, the binary will be available under
-`<example-dir>/build/zephyr/zephyr.hex`
+1. Navigate to the example's directory:
 
-### Supported nRF Connect SDK versions
+        $ cd examples/lighting-app/nrfconnect
 
-It is recommended to use the nRF Connect version which is being verified as a
-part of CHIP Continuous Integration testing, which happens to be `v1.4.0` at the
-moment. You may verify that the revision is used in
-[chip-build-nrf-platform](https://github.com/project-chip/connectedhomeip/blob/master/integrations/docker/images/chip-build-nrf-platform/Dockerfile)
-Docker image in case of doubt.
+2. Run the following command to build the example, with *board-name* replaced with the name
+   of the Nordic Semiconductor's board you own, for example `nrf52840dk_nrf52840`:
 
-Please refer to
-[this section](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#updating-the-repositories)
-in the user guide to learn how to update nRF Connect SDK repository. For example
-to checkout given `v1.4.0` revision the following commands should be called:
+        $ west build -b board-name
 
-        # Phrase <nrfconnect-dir> should be replaced with an absolute path to nRF Connect SDK source directory.
-        $ cd <nrfconnect-dir>/nrf
+   You only need to specify the board name on the first build.
 
-        $ git fetch origin
-        $ git checkout v1.4.0
-        $ west update
+3. Run the following command:
+
+        $ west build
 
-Alternatively, if you use the docker container, you may execute the following
-command instead:
+4. Remove all build artifacts by running the following command:
 
-        $ setup --ncs v1.4.0
+        $ rm -r build
+
+5. Build the example with the release configuration that disables the diagnostic features
+   like logs and command-line interface by running the following command:
+
+        $ west build -b board-name -- -DOVERLAY_CONFIG=third_party/connectedhomeip/config/nrfconnect/release.conf
+
+   Remember to replace *board-name* with the name of the Nordic Semiconductor's board you own.
+
+<hr>
 
 <a name="configuring"></a>
 
 ## Configuring the example
```

Comment:
The chapter should be removed, we can probably only link to the configuration guide here

Reply:
I'm not sure about this. The configuration page has some additional details (permanent saving, temporary saving), and the steps here give you a short overview of how to start configuration. We can, however, add a link to the Configuration page at the end, in line 308.